config/src/test/java/org/axonframework/config/AggregateConfigurerTest.java

```diff
@@ -209,6 +214,50 @@ void testAggregateConfigurationThrowsAxonConfigExceptionWhenCreatingRevisionSnap
 
         assertThrows(AxonConfigurationException.class, undefinedDeclaredAggregateTypeTestSubject::snapshotFilter);
     }
+    @Test
+    void testNullRevisionEventAndNullRevisionAggregateAllowed(){
+        DomainEventMessage<TestAggregateWithRevision> snapshotEvent = new GenericDomainEventMessage<>(
+                TestAggregateWithRevision.class.getSimpleName(), "some-aggregate-id", 0, new TestAggregateWithRevision()
```

Comment:
Shouldn't the used aggregate be the `TestAggregate` instead of the `TestAggregateWithRevision`, as the `TestAggregate` does not have a revision as opposed to the `TestAggregateWithRevision`?